index.html

```diff
@@ -37,7 +37,7 @@ <h2> As A Man Thinketh | James Allen | Full Audiobook </h2>
             <h3> Suggested videos </h3>
             <div class="film">
                 <div>
-                    <img src="https://m.media-amazon.com/images/M/MV5BMjIxODAzNTIzN15BMl5BanBnXkFtZTcwNDE2ODg5Nw@@._V1_.jpg" class="thumbnail"/>
+                    <img src="https://m.media-amazon.com/images/M/MV5BMjIxODAzNTIzN15BMl5BanBnXkFtZTcwNDE2ODg5Nw@@._V1_.jpg" class="thumbnail" alt="suggested video"/>
```

Comment:
great job, adding the alt attribute in the image tag however alt` attribute should be descriptive this is because the text is displayed when the image can't load because of slow connection, and the screen reader can read the text to visually impaired users to better understand an on-page image